components/admin/DownloadBoostWinnersButton.tsx

```diff
@@ -2,6 +2,7 @@ import React from "react";
 import Button from "@components/UI/button";
 import { useNotification } from "@context/NotificationProvider";
```

Comment:
The aim of this issue is to remove these three components, and directly use the DownloadButton you created.